Simplenote/Classes/SPNoteListViewController+Extensions.swift

```diff
@@ -630,7 +630,13 @@ private extension SPNoteListViewController {
         let cell = tableView.dequeueReusableCell(ofType: SPTagTableViewCell.self, for: indexPath)
         cell.leftImage = .image(name: .tag)
         cell.leftImageTintColor = .simplenoteNoteShareStatusImageColor
-        cell.titleText = String.searchOperatorForTags + tag.name
+        cell.titleText = String.searchOperatorForTags.tagsKeyword + tag.name
+        
+        //Display localized tag search
+        if searchPrefixIsLocalizedTag() {
+            cell.titleText = String.searchOperatorForTags.localizedTagKeyword + tag.name
+        }
```

Comment:
By the way!! I'm not sure if we need to change the way this is presented to the user. As far as I know we just need to support `Localized("tag:", ...)` and `tag:` (because users mentioned that, in some languages, just typing `tag` is easier).

Can we check with @SylvesterWilmott on this one?

Reply:
When I was setting it up I thought it was odd that I would type étiquette and I would see tag, except in the header, so I thought translating the key would be less confusing for the user.

I think I prefer removing the word tag: or translated-tag: from the results all together, The user would see the header that says "search by tag" and the tag icon next to it.  I think adding the word tag to the results is overkill.

But yes, @SylvesterWilmott WDYT

Reply:
Localising the key looks like the ideal scenario IMO. I would run it by the team as well since we will want to align with this.